### PR TITLE
Remove python .whl package dependency to swsssdk

### PR DESCRIPTION
**Pull request by liuh-80**
**- What I did**
Update setup.py to remove python .whl package dependency to swsssdk.

**- How I did it**
Update setup.py and remove dependency.

**- How to verify it**
Pass all UT.

**- Description for the changelog**
Update setup.py to remove python .whl package dependency to swsssdk.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 from setuptools import setup, find_packages
 
 dependencies = [
-    'swsssdk>=2.0.1',
```
Comment (liuh-80) [inline setup.py:4]: Need remove this line to remove .whl package dependency to swsssdk package.

**Comment by qiluo-msft** [comment]
> test_deps = [

Do you need add `swsssdk` into `test_deps`?

---
Refers to: setup.py:8 in 10d6c3e. [](commit_id = 10d6c3e623f5e725f5d7229e1b7f697535feee0b, deletion_comment = False)

File: setup.py (hunk 1 above)
Comment (liuh-80) [reply setup.py:4]: > > test_deps = [
> 
> Do you need add `swsssdk` into `test_deps`?
> 
> Refers to: setup.py:8 in [10d6c3e](https://github.com/Azure/sonic-snmpagent/commit/10d6c3e623f5e725f5d7229e1b7f697535feee0b). [](commit_id = [10d6c3e](https://github.com/Azure/sonic-snmpagent/commit/10d6c3e623f5e725f5d7229e1b7f697535feee0b), deletion_comment = False)

Will check if add to test_deps can pass build image.

**Comment by liuh-80** [comment]
> > test_deps = [
> 
> Do you need add `swsssdk` into `test_deps`?
> 
> Refers to: setup.py:8 in [10d6c3e](https://github.com/Azure/sonic-snmpagent/commit/10d6c3e623f5e725f5d7229e1b7f697535feee0b). [](commit_id = [10d6c3e](https://github.com/Azure/sonic-snmpagent/commit/10d6c3e623f5e725f5d7229e1b7f697535feee0b), deletion_comment = False)

Fixed.

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**